css/styles.css

```diff
@@ -0,0 +1,73 @@
+.body {
+  background-color: #CBFDCB;
+
+}
+.header {
+  background-color: #228B22;
+  padding: 1em;
+  text-align: center;
+}
+.header__nav {
+  font-weight: bold;
+}
+.nav__links {
+  margin: 0 .5em;
+  color: #CBFDCB;
+  font-family: 'Signika', sans-serif;
+}
+.nav__links__current {
+  margin: 0 .5em;
+  color: #FFFFFF;
+  font-family: 'Signika', sans-serif;
```

Comment:
Good use of a fallback font.  👍  Consider setting broad default styles on the `body` tag, then overwrite as necessary.